src/test/resources/CrossBrowsers.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="CrossBrowsers">
```

Comment:
Hi,
Your homework 25 have been reviewed, That's oke. You're doing great! Keep up the good work.

TestNG supports multithreading, Multithreading can be achieved by using the keyword "thread-count=<integer>" at Testng.xml. Thread count is basically a number of instances running to execute multiple tests simultaneously or in parallel. The attribute thread-count allows the user to specify how many threads should be run for this execution. 
So tf you run test in the multithreading, you can add thread-count in the testNG xml file
//Example: 
```
<suite name="CrossBrowsers" parallel="methods" thread-count="2">
  ... 
</suite>
```
Methods: Helps run methods in separate threads